git/Cargo.toml

```diff
@@ -1,8 +1,10 @@
 [package]
 name = "git"
```

Comment:
I suspect this is going to try and publish it as `git` — which is taken. Probably need to rename this project to `gitkv`?

Reply:
(And rename the other ones too to `gitkv-<sub project>`, eg. `gitkv-server`?

Reply:
not sure we want to publish the subprojects, we are only interested in publishing the server `gitkv-server`

Reply:
I don't think you can do that — I think you have to publish all of them as `gitkv-server` has a dependency on the others?

I'd have to look TBH!